setup.py

```diff
@@ -4,7 +4,7 @@
 import setuptools
 import fairlearn
 
-with open("README.md", "r") as fh:
+with open("README.md", "r", encoding="utf8") as fh:
```

Comment:
We've not hit this before.... Can I ask the Python version and OS you hit this on? I'm wondering if out gates have a gap.

Reply:
@riedgar-ms  I saved the file in Sublime Text on Catalina, the `pip install` failures were in the Azure build pipelines, no on my computer.  See the comment on the issue or you can check out those logs if you want to dig in more.

Reply:
Looks related to https://stackoverflow.com/questions/49562499/how-to-fix-unicodedecodeerror-charmap-codec-cant-decode-byte-0x9d-in-posit

As long as it doesn't affect it on other people's machines I don't mind adding this. The pipelines seem to confirm that.